src/PlatformLinux.cpp

```diff
@@ -183,6 +183,23 @@ bool Platform::dirRemove(const std::string& path) {
 	return true;
 }
 
+std::string Platform::dirGetLocal() {
+	char abs_path[1024];
+	ssize_t len = readlink("/proc/self/exe", abs_path, 1024);
+	if (len < 0 || len >= 1024)	return "./";
+	
+	// remove executable name from abs_path
+	const char BREAK_POINT = '/';
+	const char BREAK_STRING = '\0';
+	for(ssize_t i = len; i >= 0; --i) {
+		if(abs_path[i] == BREAK_POINT) {
+			abs_path[i + 1] = BREAK_STRING;
+			break;
+		}
+	}
+	return abs_path;
```

Comment:
While I think it happens implicitly, I think this should be:
```
return std::string(abs_path);
```
Better to be explicit for things like this in my opinion.

Reply:
Done